java/src/server/persistence/plugin/IPersistencePlugin.java

```diff
@@ -1,35 +1,55 @@
-package server.persistence.plugins;
-
-
-/**
- * Created by Kyle 'TMD' Cornelison on 4/2/2016.
- */
-public interface IPersistencePlugin {
-    //region Plugin Methods
-    /**
-     * Returns a connection to the database
-     *
-     * @return
-     */
-    Object getConnection(); // TODO: 4/2/2016 Maybe use a wrapper instead of Object...
-
-    /**
-     * Clears the database
-     */
-    void clear();
-
-    /**
-     * Starts a transaction on the database
-     *
-     */
-    void startTransaction();
-
-    /**
-     * Ends a transaction on the database
-     *
-     * @param commitTransaction
-     * @throws
-     */
-    void endTransaction(boolean commitTransaction);
-    //endregion
-}
+package server.persistence.plugin;
+
+import server.persistence.daos.*;
+
+/**
+ * Created by Kyle 'TMD' Cornelison on 4/2/2016.
+ */
+public interface IPersistencePlugin {
+    //region Plugin Methods
+    /**
+     * Returns a connection to the database
+     *
+     * @return
+     */
+    Object getConnection(); // TODO: 4/2/2016 Maybe use a wrapper instead of Object...
```

Comment:
In computer science, a database connection is the means by which a database server and its client software communicate with each other. The term is used whether or not the client and the server are on different machines. The client uses a database connection to send commands to and receive replies from the server.